src/components/TouchHintCircle.js

```diff
@@ -0,0 +1,102 @@
+// @flow
+
+import React, { Component } from "react";
+import {
+  InteractionManager,
+  Animated,
+  View,
+  StyleSheet,
+  Easing,
+} from "react-native";
+import colors from "../colors";
+
+class TouchHintCircle extends Component<*> {
+  leftAnimated = new Animated.Value(0);
+  opacityAnimated = new Animated.Value(1);
+
+  componentDidMount() {
+    Animated.spring(this.opacityAnimated, {
+      toValue: 0,
+      delay: 200,
+      duration: 600,
+      useNativeDriver: true,
+    }).start();
+
+    Animated.loop(
+      Animated.timing(this.leftAnimated, {
+        toValue: 1,
+        duration: 1000,
+        easing: Easing.elastic(1),
+        useNativeDriver: true,
+      }),
+      {
+        delay: 800,
+        iterations: 3,
+      },
+    ).start();
+
+    Animated.spring(this.opacityAnimated, {
+      toValue: 0,
+      delay: 3000,
+      duration: 1200,
+      useNativeDriver: true,
+    }).start();
```

Comment:
It is a better practice to regroup the 3 animated into one big `start()`

you can do that by using `Animated.parallel([...]).start()`